xbmc/music/infoscanner/MusicInfoScanner.cpp

```diff
@@ -1066,6 +1066,13 @@ void CMusicInfoScanner::FindArtForAlbums(VECALBUMS &albums, const std::string &p
   if (albums.size() == 1)
   {
     CFileItem album(path, true);
+    /* If we are scanning a directory served over http(s) the root directory for an album will set
+   * IsInternetStream to true which prevents scanning it for art.  As we can't reach this point
+   * without having read some tags (and tags are not read from streams) we can safely check for
+   * that case and set the IsHTTPDirectory property to enable scanning for art.
+   */
```

Comment:
If you increase indentation of these four lines by 2 spaces, the comment structure will look more vertically uniform.

Reply:
Sure.  It appears that if you use `/*.......*/` clang-format only applies the indent to the first line. If you use `//` for each line, it indents all of them.  I'll have to watch for that in future.